Framework/Algorithms/src/ConvertAxesToRealSpace.cpp

```diff
@@ -258,7 +258,7 @@ void ConvertAxesToRealSpace::exec() {
   // loop over the data and sqrt the errors to complete the error calculation
   PARALLEL_FOR1(outputWs)
   for (int i = 0; i < nOutputHist; ++i) {
-    MantidVec &errorVec = outputWs->dataE(i);
+    auto &errorVec = outputWs->mutableE(i);
     std::transform(errorVec.begin(), errorVec.end(), errorVec.begin(),
                    static_cast<double (*)(double)>(sqrt));
```

Comment:
Another case of the dangerous use of E as variance (and subsequent conversion), put this on the list together with `rebinHistogram`.